<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1169</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Manager</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Project:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="projects_cmb">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="compareServer_chb">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Compare with SERVER</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="refresh_btn">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="serverHelp_wg" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QFrame" name="frame_8">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:#CACAD4;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>BOTH IN ZERO</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_22">
           <item>
            <widget class="QFrame" name="frame_9">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #000000;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>FILE NOT EXISTS</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_23">
           <item>
            <widget class="QFrame" name="frame_10">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #00CC00;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>BOTH IN SINCRO</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_24">
           <item>
            <widget class="QFrame" name="frame_11">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #CC66FF;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>NEEDS UPLOAD</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_25">
           <item>
            <widget class="QFrame" name="frame_12">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #FF0000;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>NEEDS DOWNLOAD</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="assetsTab">
       <attribute name="title">
        <string>Assets</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Search:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="search_asset_le"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="assets_tw">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Model</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Shading</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Hrs</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Rig</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Final</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sequencesTab">
       <attribute name="title">
        <string>Sequences</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Search:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="search_shot_le"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QListWidget" name="sequences_lw">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
          <widget class="QTableWidget" name="shots_tw">
           <column>
            <property name="text">
             <string>Shot</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Layout</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Anim</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>SkinFix</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Hrs</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Vfx</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Sim</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Lit</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Comp</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tasks</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_11">
              <property name="styleSheet">
               <string notr="true">font: 75 23pt &quot;Corbel&quot;;
font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Tasks:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>+ Task</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="task_lw">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">font: 75 20pt &quot;Corbel&quot;;
font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Current Task:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="styleSheet">
               <string notr="true">font: 75 20pt &quot;Corbel&quot;;
font-weight: bold;</string>
              </property>
              <property name="text">
               <string>TASK NAME</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>Waiting</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Review</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Pending</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>In-Progress</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Approved</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ready</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="endDate_lbl">
              <property name="text">
               <string>End Date</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="priority_val_lbl">
              <property name="text">
               <string>Priority-VALUe</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="startDate_lbl">
              <property name="text">
               <string>Start Date</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">font: 75 10pt &quot;Corbel&quot;;
font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Start:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_7">
              <property name="styleSheet">
               <string notr="true">font: 75 10pt &quot;Corbel&quot;;
font-weight: bold;</string>
              </property>
              <property name="text">
               <string>End:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="styleSheet">
               <string notr="true">font: 75 10pt &quot;Corbel&quot;;
font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Priority:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="styleSheet">
               <string notr="true">font: 75 10pt &quot;Corbel&quot;;
font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="styleSheet">
             <string notr="true">font: 75 20pt &quot;Corbel&quot;;
font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Notes:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="reply_lbl">
              <property name="styleSheet">
               <string notr="true">font: 75 10pt &quot;Corbel&quot;;
font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Reply:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="note_te">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="note_lw"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1169</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionGlobal_Settings"/>
   </widget>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>Create</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionNew_Asset"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Sequence"/>
    <addaction name="actionNew_Shot"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCheck_Red_Items"/>
    <addaction name="actionCopy_Selected_From_Server"/>
    <addaction name="separator"/>
    <addaction name="actionReference_Selected_Items"/>
   </widget>
   <widget class="QMenu" name="menuHistory">
    <property name="title">
     <string>History</string>
    </property>
   </widget>
   <addaction name="menuCreate"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHistory"/>
  </widget>
  <action name="actionSet_Projects_Path">
   <property name="text">
    <string>Set Projects Path</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionNew_Asset">
   <property name="text">
    <string>New Asset</string>
   </property>
  </action>
  <action name="actionNew_Sequence">
   <property name="text">
    <string>New Sequence</string>
   </property>
  </action>
  <action name="actionNew_Shot">
   <property name="text">
    <string>New Shot</string>
   </property>
  </action>
  <action name="actionNew_Set">
   <property name="text">
    <string>New Set</string>
   </property>
  </action>
  <action name="actionCopy_Selected_From_Server">
   <property name="text">
    <string>Copy Selected From Server</string>
   </property>
  </action>
  <action name="actionCheck_Red_Items">
   <property name="text">
    <string>Select Red Items</string>
   </property>
  </action>
  <action name="actionReference_Selected_Items">
   <property name="text">
    <string>Reference Selected Items</string>
   </property>
  </action>
  <action name="actionGlobal_Settings">
   <property name="text">
    <string>Global Settings</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>shots_tw</tabstop>
  <tabstop>projects_cmb</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>assets_tw</tabstop>
  <tabstop>compareServer_chb</tabstop>
  <tabstop>sequences_lw</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>compareServer_chb</sender>
   <signal>toggled(bool)</signal>
   <receiver>serverHelp_wg</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
